frontend/src/SettingsPage/SettingsPage.jsx

```diff
@@ -87,11 +87,12 @@ function SettingsPage(props) {
   };
 
   const handlePasswordInput = (input) => {
-    setNewPassword(input);
-    if (input.length > 100) {
+    const sanitizedInput = input.replace(/\s/g, '');
```

Comment:
Even though adding whitespace between words is not a common practice when choosing passwords, we shouldn't remove all whitespace. Instead, we should only remove spaces from the beginning and end.

So shall we do this instead ?
`const sanitizedInput = input.trim();
`

Reply:
sure I'll change it to trim